### PR TITLE
Main candidate 20 April 21

### PR DESCRIPTION
**Pull request by 83bhp**

### CODE REVIEW

**Review by SomwanshiAnjali — APPROVED** [review]

**Review by mazinamdar — APPROVED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: srv/components/misc_pkgs/kafka/start.sls [hunk 1]
```diff
@@ -22,10 +22,18 @@ Start zoopkeper:
     - cwd: /opt/kafka
     - unless: test 1 -le $(ps ax | grep java | grep -i QuorumPeerMain | grep -v grep | awk '{print $1}' | wc -l)
 
+#TODO: find better solution to add delay
+Wait for zookeeper to start:
```
Comment (ypise) [inline srv/components/misc_pkgs/kafka/start.sls:26]: This is not a good solution.
This should be handled with a polling loop.

To be addressed against https://github.com/Seagate/cortx-prvsnr/issues/1136

File: srv/components/misc_pkgs/kafka/stop.sls [hunk 2]
```diff
@@ -15,15 +15,24 @@
...
   cmd.run:
     - name: ./bin/kafka-server-stop.sh -daemon config/server.properties
-    - cwd: /opt/kafka/kafka_{{ kafka_version }}
+    - cwd: /opt/kafka
     - onlyif: test 1 -le $(ps ax | grep ' kafka\.Kafka ' | grep java | grep -v grep | awk '{print $1}' | wc -l)
 
+#TODO: find better solution to add delay
+Wait for kafka to stop:
+  module.run:
+    - test.sleep:
```
Comment (ypise) [inline srv/components/misc_pkgs/kafka/stop.sls:27]: Not elegant.

To be addressed against #1136

**Review by ypise — APPROVED** [review]
To be addressed against #1136

**Pull request merged**